PProPanel/jsx/general.jsx

```diff
@@ -0,0 +1,6 @@
+if (typeof $ == 'undefined') {
```

Comment:
It was part of samples before, so I kept it. In general it is always better to have it checked than to get an error. Especially in Extendscript :-P 

@bbb999 is the $ object always injected? That you put "that is correct" in [ ] let's me assume that you're actually not 100% sure ;-)